ParisBNF/abb/BNFabb20.xml

```diff
@@ -52,60 +93,212 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                                        </placeName>ወ<placeName ref="LOC3930Iran">ፋርስ፡ </placeName> ወኵራስን፡
+                                        ወፌርያ፡ ወስንድ፡
+                                        ወህንዳኬ፡ ወ<placeName ref="LOC6387Yemen">የማን</placeName>፡
+                                        ወጣይፍ፡ ወተማማ፡ ወ፪አብሕርት፡ ወዘመደ፡ <placeName ref="ETH1134Arab">ዐረብ፡</placeName> ወእለ፡ የአምሩ፡ ጽሒፈ፡ ሆኅያት፡ ፳<placeName ref="ETH1678Hebrew">እብራዊ፡
+                                        </placeName>ወሥየናዊ፡ ወፋርሳዊ፡  ወካልዳዊ፡ ወሲኒ፡
+                                        ወኃማዚ፡ ወ<placeName ref="ETH1134Arab">ዐረቢ፡ </placeName>
+                                        ወሎሙ፡ እምአብሕርት፡ <placeName ref="LOC3013Euphr">ኤፍራጦስ</placeName>፡ ወፈለገ፡ በለከ፡</q>
                                     <q type="translation" xml:lang="de">
                                                 <ab>Die Zahl derer, die für den Bau des Turms versammelt wurden, waren 72
                                                     Hauptleute. Unter ihnen waren von den Nachkommen Sems 25 Mann. Diese wohnten
                                                     vom Euphrat bis Syrien und bis zum östlichen Randbereich. Zu ihnen gehören
-                                                    die Syrer, die Gebiete von Räbiʿa (= Diyar Rabia), die Insel Wäḥarāmqā (=
-                                                    Ǧarāmiqa), die Chaldäer - das sind die Leute von Babylon, Persien, Khorasan,
-                                                    Feryā (= Fergana), Sind, Indien, Jemen, Ṭaif (= aṭ-Ṭayn), Tämāmā (=
-                                                    al-Yamāma) sowie zwei Meere und die Araber. Diejenigen, die mit Buchstaben
-                                                    zu schreiben wissen, sind acht: Hebräer, Śǝyänāwi (= Syrer), Perser, Inder,
-                                                    Chaldäer, Sini (= Chinesen), Ḫāmāzi (= Himyaren) und Araber. An Gewässern
-                                                    haben sie den Euphrat und den Strom Bäläkä (= Balch).</ab>
+                                                    die Syrer, die Gebiete von Rabiʿa, die Insel Waḥarāmqā, die Chaldäer - das sind die Leute von Babylon, Persien, Khorasan,
+                                                    Feryā, Sind, Indien, Jemen, Ṭayǝf, Tamāmā sowie zwei Meere und die Araber. Diejenigen, die mit Buchstaben
+                                                    zu schreiben wissen, sind acht: Hebräer, Śǝyanāwi, Perser, Inder,
+                                                    Chaldäer, Sini, Ḫāmāzi und Araber. An Gewässern
+                                                    haben sie den Euphrat und den Strom Balaka.</ab>
                                     </q>
+                                    <note>Rabiʿa should be understood as Diyar Rabia, the island Waḥarāmqā as Ǧarāmiqa, Feryā as Fergana, Ṭayf as aṭ-Ṭayn, Tamāmā as
```

Comment:
Delete: "Ṭayf as aṭ-Ṭayn". 
This identification is outdated. Ṭayf is more likely Ṭayfa, the Arabic oasis and ancient trading centre.